pages/establishment-address.js

```diff
@@ -58,6 +58,7 @@ const EstablishmentAddress = (props) => (
               touched: true,
               error: props.t(props.validatorErrors.establishment_postcode_find)
             }}
+            hint={props.t("For example, SW1H 9EX")}
```

Comment:
Any reason why you chose that example postcode?

Reply:
FSA HQ's postcode

Reply:
Great :)